sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -222,6 +225,8 @@ def __init__(
         batch will be fed into the inference_fn as a Sequence of Tensors.
       max_batch_size: the maximum batch size to use when batching inputs. This
         batch will be fed into the inference_fn as a Sequence of Tensors.
+      load_model_args: a dictionary of parameters passed to the torch.load
+          function of to specify custom config for loading models.
```

Comment:
nit: 
```suggestion
          function to specify custom config for loading models.
```

Same below